File: src/main/java/com/codingforcookies/betterrecords/api/wire/IRecordWire.java
package com.codingforcookies.betterrecords.api.wire;

import com.codingforcookies.betterrecords.api.connection.RecordConnection;

import java.util.ArrayList;

public interface IRecordWire {
    ArrayList<RecordConnection> getConnections();

    String getName();
    float getSongRadiusIncrease();
}

File: src/main/kotlin/com/codingforcookies/betterrecords/api/wire/IRecordWire.kt
package com.codingforcookies.betterrecords.api.wire

import com.codingforcookies.betterrecords.api.connection.RecordConnection

interface IRecordWire {

    val connections: MutableList<RecordConnection>

    fun getName(): String

    val songRadiusIncrease: Float
}
